docs/Benodigde-tabellen.md

```diff
@@ -0,0 +1,20 @@
+Noot: De landelijke tabellen zijn gepubliceerd op https://publicaties.rvig.nl/Landelijke_tabellen/Landelijke_tabellen_32_t_m_61_excl_tabel_35
+
+Voor de BRP-Personen-Bevragen moeten minimaal de volgende tabellen beschikbaar worden gemaakt:
+
+Aanduiding naamgebruik (Waardelijst uit het LO GBA Element 61.10 Aanduiding naamgebruik)
```

Comment:
hier noem je de tabel "Aanduiding naamgebruik". In tabelwaarden,feature heet deze "naamgebruik"

Reply:
"Waardelijst uit het LO GBA Element": in het LO GBA heet dit "Mogelijke waarden". In de BRP API hebben wij daar het technische component "Waardetabel" aan gehangen. De term "Waardelijst" kennen we denk ik niet.

Reply:
Aangepast